doc/api/debugger.md

```diff
@@ -6,17 +6,18 @@
 
 Node.js includes an out-of-process debugging utility accessible via a
 [TCP-based protocol][] and built-in debugging client. To use it, start Node.js
-with the `debug` argument followed by the path to the script to debug; a prompt
+with the `inspect` argument followed by the path to the script to debug; a prompt
 will be displayed indicating successful launch of the debugger:
 
 ```txt
-$ node debug myscript.js
-< Debugger listening on 127.0.0.1:5858
-connecting to 127.0.0.1:5858 ... ok
-break in /home/indutny/Code/git/indutny/myscript.js:1
-> 1 global.x = 5;
+$ node inspect myscript.js
+< Debugger listening on ws://127.0.0.1:9229/80e7a814-7cd3-49fb-921a-2e02228cd5ba
+< For help see https://nodejs.org/en/docs/inspector
+< Debugger attached.
+Break on start in myscript.js:1
+> 1 (function (exports, require, module, __filename, __dirname) { global.x = 5;
   2 setTimeout(() => {
-  3   debugger;
+  3   console.log('world');
```

Comment:
Yeah, I wasn't sure about the exact "order" here. Because it first shows this and then later/below says:

> Inserting the statement `debugger;` into the source code of a script will
enable a breakpoint at that position in the code:

I interpreted that as "this is showing the original script run, then you insert `debugger`, then you run it again".